### PR TITLE
Improve README me to make it easier to get started with "Aruba" for beginners

### PR DESCRIPTION
**Pull request by maxmeyer**
## Summary

This PR tries to improve the README.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I've added tests for my code
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by maxmeyer** [comment]
FYI @mattwynne Support page on cucumber.io is referenced.

File: README.md [hunk 1]
```diff
@@ -33,11 +33,23 @@ $ gem install aruba
...
-### Release Policy
+## Support
+
+### Channels
+
+For support, please have a look at the ["support website"](https://cucumber.io/support) of "Cucumber". You have different options to reach out for help: Recommended for "aruba" are using the "Slack" channels - e.g. "aruba-devs" or "general" ([Register account](https://cucumberbdd-slack-invite.herokuapp.com/)) and the "Issues"-page on [GitHub](https://github.com/cucumber/aruba/issues). Adressing one of the matainers might directly would be helpful.
+
+### Maintainers
+
+Currently, this gem is mainly maitained by this group of people:
```
Comment (olleolleolle) [inline README.md]: Typo "maitained" => "maintained"

File: README.md [hunk 2]
```diff
@@ -33,11 +33,23 @@ $ gem install aruba
 
 Our most current documentation to get started with `aruba`, can be found on [![See our documentation on Cucumber Pro](https://github.com/cucumber-ltd/brand/raw/master/images/png/notm/cucumber-pro-black/cucumber-pro-black-32.png)](https://app.cucumber.pro/projects/aruba). It's based on our feature files describing the use of `aruba`.
 
-### Release Policy
+## Support
+
+### Channels
+
+For support, please have a look at the ["support website"](https://cucumber.io/support) of "Cucumber". You have different options to reach out for help: Recommended for "aruba" are using the "Slack" channels - e.g. "aruba-devs" or "general" ([Register account](https://cucumberbdd-slack-invite.herokuapp.com/)) and the "Issues"-page on [GitHub](https://github.com/cucumber/aruba/issues). Adressing one of the matainers might directly would be helpful.
```
Comment (olleolleolle) [inline README.md]: This last sentence may need some polish.

**Comment by maxmeyer** [comment]
@olleolleolle Thanks for the feedback. Can you send me a PR with your suggestions, please.

**Pull request merged**